util/gui.py

```diff
@@ -11,43 +11,47 @@ class GuiUtils(object):
     Class containing utility methods for interacting with the gui repository.
     """
 
-    def __init__(self, path):
+    def __init__(self, path: str) -> None:
         self.git = GitUtils(path)
         self.path = path
 
-    def get_gui_repo_at_release(self, version):
-        version = VersionUtils.extract_release_numbers_from_string(version)
-        branch_name = VersionUtils.convert_release_to_tag_name(*version)
+    def get_gui_repo_at_release(self, version_str: str) -> str | None:
+        version: list[int] = VersionUtils.extract_release_numbers_from_string(version_str)
+        branch_name: str = VersionUtils.convert_release_to_tag_name(*version)
         if not self.git.update_branch(branch_name, True):
             raise IOError(
                 "Couldn't check out GUI branch corresponding to release {}".format(version)
             )
```

Comment:
Not quite sure about this, the functions never returns so I think it's just `-> None`?

Reply:
Quite right - missed that.